examples/get-started/first_figure.py

```diff
@@ -0,0 +1,156 @@
+"""
+1. Making your first figure
+===========================
+
+This tutorial page covers the basics of creating a figure. It will only use
+the ``coast`` module for plotting. Later examples will address other PyGMT
+modules.
+"""
+# sphinx_gallery_thumbnail_number = 4
+
```

Comment:
In the Vercel preview, there's an odd empty code block here (https://pygmt-55t3spnwe-gmt.vercel.app/get-started/first_figure.html#sphx-glr-get-started-first-figure-py), in the spot where the import statement goes.

Reply:
I want to still explain `import pygmt` rather than start the tutorial with it. Any idea how to remove the empty code block?

Reply:
The latest two code suggestions should work for this.